src/styles/settings.scss

```diff
@@ -8,6 +8,8 @@ $darkGray: #2a2a2a;
 $fuelYellow: #d58e32;
 $black: #292929;
 $white: #fff;
+$green: #2ed573;
```

Comment:
Myślę, że dobrze by było przy nazwach kolorów dodawać "dark" lub "light".

Reply:
dzięki jak będę robił funkcjonalność to zajrzę do tego